### PR TITLE
refactor: adds unit test to tls

### PR DESCRIPTION
**Pull request by madhavilosetty-intel**

### CODE REVIEW

File: internal/local/mebx.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	// Set up MEBx with the provided password.
+	response, err := service.interfacedWsmanMessage.SetupMEBX(service.flags.MEBxPassword)
+	// log.Info(response.JSON())
+	if err != nil {
+		log.Error("Failed to configure MEBx Password:", err)
+		return err
+	}
+
+	// Check the response's success status.
+	if response.Body.SetMEBxPassword_OUTPUT.ReturnValue != 0 {
```
Comment (rsdmike) [inline internal/local/mebx.go:33]: we have generally been putting this check in go-wsman-messages, can you move inside of the SetupMEBX call in go-wsman-messages?

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: internal/local/mebx.go (hunk 1 above)
Comment (madhavilosetty-intel) [reply internal/local/mebx.go]: done

File: internal/local/tls.go [hunk 2]
```diff
@@ -128,30 +129,26 @@ func (service *ProvisioningService) EnableTLS() error {
...
 			}
 		}
-		if err != nil {
-			return err
-		}
 	}
 	// service.Pause(service.flags.ConfigTLSInfo.DelayInSeconds)
 	// time.Sleep(time.Duration(howManySeconds) * time.Second)
 	commitResponse, err := service.interfacedWsmanMessage.CommitChanges()
+	log.Trace(commitResponse)
```
Comment (rsdmike) [inline internal/local/tls.go:146]: please remove

File: internal/local/tls.go [hunk 3]
```diff
@@ -160,6 +157,7 @@ func (service *ProvisioningService) ConfigureTLSSettings(setting tls.SettingData
 	log.Trace(putResponse)
```
Comment (rsdmike) [inline internal/local/tls.go:157]: lets remove the trace message here as well

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: internal/local/tls.go (hunk 2 above)
Comment (madhavilosetty-intel) [reply internal/local/tls.go]: done

File: internal/local/tls.go (hunk 3 above)
Comment (madhavilosetty-intel) [reply internal/local/tls.go]: done

**Review by rsdmike — APPROVED** [review]

**Pull request merged**